packages/open-truss/src/uqi/example/README.md

```diff
@@ -0,0 +1,7 @@
+# Unified Query Interface Example
+
+First check out the fake-client implementation and tests.
+
+Then check out the fake-client-uqi implementation and tests.
+
+We'll be providing more example implementations for Trino, Kusto, MySQL, REST, and more in the future.
```

Comment:
👍 

Are you thinking that OT will have its own implementations of all those clients? I know you want to avoid the maintenance overhead of that, but it could be nice to have that open sourced. Perhaps we can set expectations around support for keeping those clients updated and recommending people implement their own or consider contributing to OT?

Reply:
I'm thinking the demo-app will have implementations of all of these clients to start and then we can pull them out into packages. We need to figure out our story around publishing many packages first though.